DeviceAdministration/Infrastructure/BusinessLogic/DeviceLogicND.cs

```diff
@@ -1156,14 +1156,6 @@ private static IEnumerable<DevicePropertyMetadata> GetDevicePropertyConfiguratio
                 PropertyType = PropertyType.Status
             };
 
-            yield return new DevicePropertyMetadata()
-            {
-                IsDisplayedForRegisteredDevices = false,
-                IsDisplayedForUnregisteredDevices = false,
-                IsEditable = false,
-                Name = "HostName"
```

Comment:
good catch, seems like it's still used in other classes. Not sure if it's a deprecated field. Adding it back in case it's still used.